staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -379,6 +387,12 @@ func NewConfig(codecs serializer.CodecFactory) *Config {
 		APIServerID:           id,
 		StorageVersionManager: storageversion.NewDefaultManager(),
 		TracerProvider:        tracing.NewNoopTracerProvider(),
+
+		// Default is to advertise aggregated discovery in the /apis
+		// endpoint if the feature is enabled Some apiservers may not
+		// want this behavior, such as aggregator apiserver, which has
+		// its own discovery implementation.
+		EnableAggregatedDiscoveryEndpoint: true,
```

Comment:
This default should depend on the featuregate.

Reply:
Hm, we probably don't need this then and can just depend on the feature gate.